custom/ewsghana/reports/__init__.py

```diff
@@ -287,7 +288,7 @@ def report_context(self):
 
     def get_report_context(self, data_provider):
         total_row = []
-        headers = []
+        headers = DataTablesHeader()
```

Comment:
It occurs when someone clicks export table before hitting apply in filters section. In EWS users don't have to click apply since we are auto loading all reports. In this case I guess someone manually modified url and get report not auto loaded and he clicked export before applying filters. It's not the best fix but I think for now it is better to generate empty export instead of raising 500.